docs/src/docs/next-steps.md

```diff
@@ -0,0 +1,232 @@
+# Next Steps with Kit
+
+In this guide you'll learn how to:
+* Make your own Kitfile
+* The power of `unpack`
+* Read the Kitfile or manifest from a ModelKit
+* Tag ModelKits and keep your registry tidy
+
+## Making your own Kitfile
+
+A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are four main parts to a Kitfile:
+1. ModelKit metadata in the `package` section, including the author, description, and license
+1. Path to the Jupyter notebook folder in the `code` section
+1. Path to the datasets in the `datasets` section (you can have multiple datasets in the same page)
```

Comment:
`multiple datasets in the same page` -> `multiple datasets in the same ModelKit`

Reply:
done